client/src/main/java/com/kritsit/casetracker/client/domain/services/Editor.java

```diff
@@ -109,7 +109,7 @@ public Staff getUser() {
 
     public List<Person> getComplainants() {
         List<Person> complainants = new ArrayList<>();
-        for (Case c : cases) {
+        for (Case c : getCases()) {
```

Comment:
"Field not initialized in constructor but dereferenced without null check." I'm not fully aware of the execution path of this program, I'm sure it is getCases() is called to initialize at some point before these are reached but that isn't good enough for FindBug.